ibm/mas_devops/playbooks/oneclick_add_predict.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Add PRedict 8.6.x application to an existing MAS Core 8.8.x + IoT 8.4.x installation
+#
+# Dependencies:
+# - ansible-playbook ibm.mas_devops.oneclick_core
+# - ansible-playbook ibm.mas_devops.oneclick_manage
+# - ansible-playbook ibm.mas_devops.oneclick_iot
+# - ansible-playbook ibm.mas_devops.oneclick_monitor
+#
+# Predict will be configured to use the same db2 instance that IoT is using
+#
+- hosts: localhost
+  any_errors_fatal: true
+
+  vars:
+    # Application Installation
+    mas_app_id: predict
+    mas_app_channel: "{{ lookup('env', 'MAS_APP_CHANNEL') | default('8.6.x', true) }}"
+
+    # Application Configuration
+    mas_workspace_id: "{{ lookup('env', 'MAS_WORKSPACE_ID') | default('masdev', true) }}"
+
+  pre_tasks:
+    # For the full set of supported environment variables refer to the playbook documentation
+    - name: Check for required environment variables
+      assert:
+        that:
+          - lookup('env', 'MAS_INSTANCE_ID') != ""
+          - lookup('env', 'MAS_ENTITLEMENT_KEY') != ""
+          - lookup('env', 'WML_INSTANCE_ID') != ""
+          - lookup('env', 'WML_URL') != ""
+          - lookup('env', 'WML_VERSION') != ""
+          - lookup('env', 'WS_USER') != ""
+          - lookup('env', 'WS_PASSWORD') != ""
+          - lookup('env', 'WS_URL') != ""
+        fail_msg: "One or more required environment variables are not defined"
+
+  roles:
+    - ibm.mas_devops.gencfg_watsonstudio
```

Comment:
This playbook should assume that CP4D is not already installed.  Oneclick playbooks need to cover all dependencies required by the app being added.  e.g. Manage adds db2, IoT adds Db2 and Kafka.